fabfile.py

```diff
@@ -1,17 +1,29 @@
-from fabric.api import *
-from fabric.colors import *
+from fabric.api import *  # noqa
+from fabric.colors import *  # noqa
 
 env.colorize_errors = True
-env.hosts           = ['sanaprotocolbuilder.me']
-env.user            = 'root'
-env.virtualenv      = 'source /usr/local/bin/virtualenvwrapper.sh'
-env.project_root    = '/opt/sana.protocol_builder'
+env.hosts = ['sanaprotocolbuilder.me']
+env.user = 'root'
+env.virtualenv = 'source /usr/local/bin/virtualenvwrapper.sh'
+env.project_root = '/opt/sana.protocol_builder'
+
 
 def test():
     local('python src/manage.py syncdb --noinput')
```

Comment:
Let's add some coloured print statements here too to make it clearer what it's doing